### ISSUE TITLE
raw BGC parameters (current and historical : DOXY, NITRATE, pH, CHLA) with QC=3  (NO QC=1 for these raw parameters)

### ISSUE THREAD

**Issue by catsch** [OneArgo/ADMT#56]

**Comment by jlovellcsiro** [comment]
CSIRO is doing this

**Comment by Argo-JMA** [comment]
Done for Doxy and Nitrate. JMA will introduce CHLA and pH RTQC by ADMT-26.

**Comment by trana99** [comment]
MEDS started to do this for doxy, chla, nitrate and PH.

**Comment by cgrdn** [comment]
Was surprised to see MEDS still has some DOXY_QC=1. I'll take a look next week and fix that.

**Comment by kamwal** [comment]
Do you have a list of the profiles which should be corrected?

**Comment by trana99** [comment]
@kamwal All of the BGC profiles (DOXY, Chlorophyll-A, Nitrate, PH, BBP700) should be corrected before sending them on the GTS

**Comment by sebartoloni** [comment]
We have compiled a list of floats that still have raw DOXY and PH_IN_SITU_TOTAL samples marked as QC = 1 on the gdac. I have attached an excel sheet below listing each WMO and its corresponding dac. 

If possible, could each dac take a look at their floats listed below and update the QC flags from 1 to 3 before ADMT-26?

Let me know if you have any questions or have trouble accessing the excel sheet.

[SUMMARY_QC1_floats_ALL_DACS_August2025.xlsx](https://github.com/user-attachments/files/21977494/SUMMARY_QC1_floats_ALL_DACS_August2025.xlsx)

** note KIOST is listed as KORDI

**Comment by Argo-JMA** [comment]
> > Done for Doxy and Nitrate. JMA will introduce CHLA and pH RTQC by ADMT-26.
> 
> [@Argo-JMA](https://github.com/Argo-JMA) Thank you! Nitrate looks great. Is DOXY done for all past and future profiles? We are still seeing two floats with QC = 1 on the gdac (listed in the attached excel sheet).

@sebartoloni Thank you for your comment. I'll check and get back to you later.

**Comment by planktonhead** [comment]
There was one CSIRO float in that excel table; it's now been addressed.

**Comment by kamwal** [comment]
> > Do you have a list of the profiles which should be corrected?
> 
> [@kamwal](https://github.com/kamwal) We just ran an audit listing all the floats that still have raw data with QC = 1. I attached our output for pH and doxy (above) but I can provide a list for other bgc parameters if requested.

HI Sarah, Could you provide us the list of the floats/profiles also of the remaining parameters please :)

**Comment by cgourcuf** [comment]
@tcarval can you please update the status for Coriolis? There are several Coriolis floats in the spreadsheet above (August 2025).

**Comment by catsch** [comment]
@cgourcuf @sebartoloni @tcarval 
the Coriolis files were updated in August

**Comment by tcarval** [comment]
DOXY
Float 6901021 : the data mode is "D", DOXY_ADJUSTED_QC = 1 is valid
Float 6901657 : idem

PH_IN_SITU_TOTAL
3901669 ph_in_situ_total_qc set to 3 is valid
...

**Comment by kyunghee-kiost** [comment]
Changed raw DOXY_QC to 3 and updated it. (KIOST)

**Comment by catsch** [comment]
Thanks to @sebartoloni, all the DACs are aware of this now 
a QC audit should be released regularly to stay in line with the recommendations 
but I think we can close it

**Comment by sebartoloni** [comment]
@catsch I agree! We will work on releasing a QC audit before ADMT. I will close this ticket now.
